components/contribution-flow/ContributionFlowStepContainer.js

```diff
@@ -132,6 +132,7 @@ class ContributionFlowStepContainer extends React.Component {
             onChange={this.props.onChange}
             data={stepDetails}
             showFeesOnTop={this.props.showFeesOnTop}
+            LoggedInUser={LoggedInUser}
```

Comment:
:ice_cream: (No blocker) It's usually better to use `useUser` or `withLoggedInUser` rather than passing down the prop

Reply:
Thanks. Done. 👍🏽